### ISSUE TITLE
Uncaught ReferenceError: rotator is not defined

### ISSUE THREAD

**Issue by WebShapedBiz** [digitalfortress-tech/ad-rotator#25]
### What happened?

Hi.

I installed ad-rotator per instructions, using the CDN version since I'm building the site using SSG, and all I can get is an error in the console saying "Uncaught ReferenceError: rotator is not defined" pointing to instance line: "const instance = rotator(..."

I've tried with both jsdelivr and unpkg, it made no difference. I can see in DevTools that the ad-rotator script is loaded, I put it in the <head> and init script before </body>. What am I doing wrong?

Thank you for your time and effort.

EDIT: I just found a fiddle that has the same error: https://jsfiddle.net/niketpathak/g3yrnjdo/

### Affected version

Latest

### Reproduction URL

https://morskitv.netlify.app/video/dropping-a-chocolate-on-a-filled-up-glass/

### How to reproduce?

I just followed the instructions and that led me to this issue.

### Browsers

Chrome

**Comment by niketpathak** [comment]
Hello, sorry for the delayed response. There is a small error in the documentation that has been fixed in [v5.8.1](https://github.com/digitalfortress-tech/ad-rotator/releases/tag/v5.8.1). 
The library needs to be initialised in a slightly different way, 

```js
// before
const instance = rotator(/* config options */); // does not work after v5.6.x

// after
const instance = rotator.init(/* config options */) 
```

The [js fiddle](https://jsfiddle.net/niketpathak/g3yrnjdo/) has been updated. Sorry about that. 
Hope this helps.

**Comment by WebShapedBiz** [comment]
Thank you @niketpathak for your time and effort.
